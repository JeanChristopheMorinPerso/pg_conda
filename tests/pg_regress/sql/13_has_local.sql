BEGIN;
CREATE EXTENSION pg_conda;

SELECT conda_has_local('1.0.0') = FALSE;
SELECT conda_has_local('1.0.0'::condaversion) = FALSE;
SELECT conda_has_local('1.0.0+whatever') = TRUE;
SELECT conda_has_local('1.0.0+whatever'::condaversion) = TRUE;
ROLLBACK;
