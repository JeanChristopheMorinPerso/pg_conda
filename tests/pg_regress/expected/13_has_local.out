BEGIN;
CREATE EXTENSION pg_conda;
SELECT conda_has_local('1.0.0') = FALSE;
 ?column? 
----------
 t
(1 row)

SELECT conda_has_local('1.0.0'::condaversion) = FALSE;
 ?column? 
----------
 t
(1 row)

SELECT conda_has_local('1.0.0+whatever') = TRUE;
 ?column? 
----------
 t
(1 row)

SELECT conda_has_local('1.0.0+whatever'::condaversion) = TRUE;
 ?column? 
----------
 t
(1 row)

ROLLBACK;
